Spokestack/SpeechConfiguration.swift

```diff
@@ -131,4 +131,62 @@ import Foundation
+    /// - Remarks: Both the file name and the file path are configurable to allow for flexibility in constructing the path that the recognizer will attempt to load the model from.
+    /// - SeeAlso: `TFLiteKeywordRecognizer`
+    @objc public var keywordFilterModelName: String = "KeywordFilter"
+    /// The filename of the machine learning model used for the encoding step of the keyword recognizer.
+    /// - Remarks: Both the file name and the file path are configurable to allow for flexibility in constructing the path that the recognizer will attempt to load the model from.
+    /// - SeeAlso: `TFLiteKeywordRecognizer`
+    @objc public var keywordEncodeModelName: String = "KeywordEncode"
+    /// The filename of the machine learning model used for the detect step of the keyword recognizer.
+    /// - Remarks: Both the file name and the file path are configurable to allow for flexibility in constructing the path that the recognizer will attempt to load the model from.
+    /// - SeeAlso: `TFLiteKeywordRecognizer`
+    @objc public var keywordDetectModelName: String = "KeywordDetect"
+    /// The filesystem path to the machine learning model for the filtering step of the keyword recognizer.
+    /// - SeeAlso: `TFLiteKeywordRecognizer`
+    @objc public var keywordFilterModelPath: String = "KeywordFilter.tflite"
+    /// The filesystem path to the machine learning model for the encoding step of the keyword recognizer.
+    /// - SeeAlso: `TFLiteKeywordRecognizer`
+    @objc public var keywordEncodeModelPath: String = "KeywordEncode.tflite"
+    /// The filesystem path to the machine learning model for the detect step of the keyword recognizer.
+    /// - SeeAlso: `TFLiteKeywordRecognizer`
+    @objc public var keywordDetectModelPath: String = "KeywordDetect.tflite"
+    /// The threshold of the keyword recognizer's posterior output, above which the keyword recognizer emits a recognition event for the most probable keyword.
+    /// - SeeAlso: `TFLiteKeywordRecognizer`
+    @objc public var keywordThreshold: Float = 0.5
+    /// A comma-separated list of keywords to recognize.
+    /// - Remark: ex: "yes,no"
+    /// - Warning: cannot contain spaces
+    /// - SeeAlso: `TFLiteKeywordRecognizer`
+    @objc public var keywords: String = "bed,bird,cat,dog,down,eight,five,four,go,happy,house,left,marvin,nine,no,off,on,one,right,seven,sheila,six,stop,three,tree,two,up,wow,yes,zero"
+    /// The name of the window function to apply to each audio frame before calculating the STFT.
+    /// - Remark: Currently the "hann" window is supported.
+    /// - SeeAlso: `TFLiteWakewordRecognizer`
+    public var keywordFFTWindowType: SignalProcessing.FFTWindowType = .hann
+    /// The desired linear Root Mean Squared (RMS) signal energy, which is used for signal normalization and should be tuned to the RMS target used during wakeword model training.
+    /// - SeeAlso: `TFLiteWakewordRecognizer`
+    @objc public var keywordRMSTarget: Float = 0.08
```

Comment:
we don't do rms normalization for keyword models, and the feature is deprecated for wakeword models, in favor of automatic gain control

Reply:
missed that change